src/azure-cli/azure/cli/command_modules/aro/custom.py

```diff
@@ -0,0 +1,150 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import random
+import os
+
+import azure.mgmt.redhatopenshift.models as v2020_04_30
+from azure.cli.command_modules.aro._aad import AADManager
+from azure.cli.command_modules.aro._rbac import assign_contributor_to_vnet
+from azure.cli.command_modules.aro._validators import validate_subnets
+from azure.cli.core.commands.client_factory import get_subscription_id
+from azure.cli.core.util import sdk_no_wait
+
+
+FP_CLIENT_ID = 'f1dd0a37-89c6-4e07-bcd1-ffd3d43d8875'
```

Comment:
Why it's a staic hard code client ID? Will is cause security issue?

Reply:
No, it will no. This is Azure FirstParty ID. It is given only for services it is guaranteed to be same and same across all clouds.